doc/contributing.md

```diff
@@ -1 +1,54 @@
 # Contributing
+Thank you for interest in contributing to LivePeer ecosystem.
+
+# Working on Go code
+If your goal is making changes solely to Go part of the codebase, just follow [Requirements](https://github.com/livepeer/lpms/#requirements) section and make sure you have all dependencies to build Go scripts successfully. Debugging with Delve, GDB, or, visually, with IDE (e.g. JetBrains GoLand, Microsoft VS Code) should work fine.
+
+# Working on CGO interface and FFMpeg code
+When working on CGO interface, or [FFmpeg](https://github.com/livepeer/FFmpeg/) fork, chances are you might want to create a separate debug build of all components. 
+
+To do it:
+1. Copy `~/compiled` dir created by `install_ffmpeg.sh` to `compiled_debug`
+2. If you want to keep system-wide Ffmpeg installation, remove/rename system `pkg-config` (.pc) files of Ffmpeg libs (located in `/usr/local/lib/pkgconfig/` on most Linux distros). Otherwise, `pkg-config` will prioritize system libs without debug information.
+3. [FFmpeg](https://github.com/livepeer/FFmpeg/) should be already cloned after running `install_ffmpeg.sh`. Navigate to FFmpeg dir and run `configure` as below, making sure all paths are correct.
+    ```
+    export ROOT=/projects/livepeer/src
+    export PATH="$ROOT/compiled_debug/bin":$PATH
+    export PKG_CONFIG_PATH="${PKG_CONFIG_PATH:-}:$ROOT/compiled_debug/lib/pkgconfig"
+    ./configure --fatal-warnings \
```

Comment:
I wonder if we can incorporate this into the `install_ffmpeg.sh` script. Would be cool if we could do something like `./install_ffmpeg.sh --debug` to do an install with debugging enabled and with the relevant libs installed stored in a dedicated debug directory like `compiled_debug`.

Reply:
+1 on having the debug flags become a part of the `install_ffmpeg.sh` script. We also need to bring this and the `go-livepeer` install ffmpeg script closer to feature parity, with the only difference being explicitly enabling only the protocols, decoders, encoders we need in `go-livepeer` because it's a binary that we release.

But this is a great starting point, and we can handle that in a separate PR.